app/build.gradle

```diff
@@ -48,4 +48,8 @@ dependencies {
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.0.1'
     androidTestImplementation 'com.jayway.android.robotium:robotium-solo:5.3.1'
 

```

Comment:
idk if it's just on my machine, but without these lines in the build.gradle
```
    testImplementation 'junit:junit:4.12'
    androidTestImplementation 'androidx.test.ext:junit:1.1.1'
    androidTestImplementation 'androidx.test:runner:1.2.0'
    androidTestImplementation 'androidx.test:rules:1.2.0'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
```
the tests complain about import errors

Reply:
before the UI tests we also have to ensure the user is logged out/on the login screen bc the tests are failing for me when I'm logged in